src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -36,22 +36,21 @@ class ThrowMatcher implements MatcherInterface
      * @var PresenterInterface
      */
     private $presenter;
-
```

Comment:
The other properties don't have an empty line between them, so just for consistency I removed this one